.codecov.yml

```diff
@@ -0,0 +1,8 @@
+coverage:
+  status:
+    project:
+      default:
+        informational: true
```

Comment:
Just to understand things better; "informational" means this is just providing the result without gating anything?